android/templates/src/main/java/com/badoo/ribs/template/rib_with_view/foo_bar/FooBarNode.kt

```diff
@@ -6,7 +6,7 @@ import com.badoo.ribs.template.rib_with_view.foo_bar.feature.FooBarFeature
 import io.reactivex.ObservableSource
 import io.reactivex.functions.Consumer
 
-class FooBarNode(
+class FooBarNode internal constructor(
```

Comment:
This file is actually a mistake, I didn't notice commiting. Can you please remove the whole subfolder structure? Templates should only contain /snapshot and /release-0.xyz subfolders at this point. Thanks.